prompting/api/gpt_endpoints/mixture_of_miners.py

```diff
@@ -0,0 +1,45 @@
+import copy
+
+from prompting.api.gpt_endpoints.process_completions import process_completions
+
+DEFAULT_SYSTEM_PROMPT = """You have been provided with a set of responses from various open-source models to the latest user query.
```

Comment:
If you have tested this thoroughly ignore this comment: To me putting this into the system prompt feels a little strange. My understanding is that this should simply be appended as a user message, but again I could be wrong on this so feel free to ignore

Reply:
It's based on original paper:
https://arxiv.org/pdf/2406.04692